include/z64actor.h

```diff
@@ -214,6 +214,11 @@ if neither of the above are set : blue
 0x2000 : translucent, else opaque
 */
 
+#define DYNAPOLYACTOR_INTERACT_FLAG_ACTOR_ON_TOP (1 << 0) // There is an actor standing on the collision of the dynapoly actor
+#define DYNAPOLYACTOR_INTERACT_FLAG_PLAYER_ON_TOP (1 << 1) // The player actor is standing on the collision of the dynapoly actor
+#define DYNAPOLYACTOR_INTERACT_FLAG_PLAYER_ABOVE (1 << 2) // The player is directly above the collision of the dynapoly actor (any distance above)
+#define DYNAPOLYACTOR_INTERACT_FLAG_3 (1 << 3) // Like DYNAPOLYACTOR_INTERACT_FLAG_ACTOR_ON_TOP but only actors with ACTOR_FLAG_26
```

Comment:
Figs PR:
`ACTOR_FLAG_26` -> `#define ACTOR_FLAG_CAN_HOLD_SWITCH (1 << 26) // actor can press and hold down switches`

So I wonder if you could name this flag something like `DYNAPOLYACTOR_INTERACT_FLAG_ACTOR_CAN_HOLD_SWITCH`? but that name is quite the handful... Or maybe you're waiting for names to be finalized?

Reply:
my rudimentary docs i posted in discord last month were along the lines of 
```
DYNA_ACTOR_ON_TOP
DYNA_PLAYER_ON_TOP
DYNA_PLAYER_OVER_TOP
DYNA_SWITCH_ACTOR_ON_TOP
```
with the variable being `/* 0x160 */ u8 topFlags; // information about what actors are on top or above `
interact might be a little tricky, cause 15C the other set of flags are also kinda about the interaction player has with the surface hes standing on, but its more about movement/transforming 

if we stick with interact flags, I think we can shorten the prefix to `DYNA_INTERACT_` tho

Reply:
I'd rather wait for the actor flag documentation to name this one tbh

Reply:
For shortening the prefix you mean like
`DYNAPOLYACTOR_INTERACT_FLAG_ACTOR_ON_TOP` -> `DYNA_INTERACT_ACTOR_ON_TOP`
?